keps/sig-cluster-lifecycle/kubeadm/20190417-v1beta2-config.md

```diff
@@ -0,0 +1,157 @@
+---
+title: “Upgrading kubeadm’s config to v1beta2”
```

Comment:
I don't think we really need a kep for this vs. creating a master KEP and outlining updates as we roll.

Reply:
You mean something like a config GA KEP, which we'll update from time to time until we are happy with the config? Thus making v1beta2 a milestone to config GA?